### ISSUE TITLE
404 page is not found for /guide/en/elasticsearch/client/java-api/current/scrolling.html

### ISSUE THREAD

**Issue by sylvie777** [elastic/elasticsearch#68824]
Reported by @dadoonet (Original ticket: https://github.com/elastic/website-www.elastic.co/issues/5994)

Provide the following information to help us identify and troubleshoot the issue.
- Page URL: https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/scrolling.html
- Specifics (e.g., OS, Hardware, Browser, etc.): Browser with FR locale

List out steps on how to recreate the issue.
1. Try with a FR browser to open the page which does not exist
2. You'll be redirected to https://www.elastic.co/fr/opt/node_apps/new404.html which does not seem to exist.

![image](https://user-images.githubusercontent.com/274222/79765647-296a0a80-8327-11ea-8fcd-889588576238.png)

**Comment by jrodewig** [comment]
It looks like this link was changed to https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/java-search-scrolling.html with the 2.0 release.

We should add a redirect so users with the stale link can get to the right page.

**Comment by dadoonet** [comment]
Sure. My concern though is that a 404 page should be found in all case... 

For example: https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/foobar.html should show:

![image](https://user-images.githubusercontent.com/274222/107522101-9fd84180-6bb3-11eb-9472-714d8dad383c.png)


Instead I have:

```
Moved Permanently. Redirecting to /fr/opt/node_apps/new404.html
```

**Comment by jrodewig** [comment]
Thanks for clarifying @dadoonet.

@sylvie777 @gtback I was able to recreate by setting my browser locale to `FR`. For some reason, it looks like this doesn't trigger our custom 404 page.

<img width="1381" alt="Screen Shot 2021-02-10 at 9 36 31 AM" src="https://user-images.githubusercontent.com/40268737/107524205-87e8c980-6b83-11eb-8f4a-dd1bd217e477.PNG">

Since this isn't a content issue, let me know which team should own this. I'll transfer to the appropriate repo.

**Comment by gtback** [comment]
I don't think this (browser locale identification and redirection) is something the docs code does. Possibly something at the web host/proxy level? We also don't have any `fr` docs, so all of these redirects are expected to fail. `opt/node_apps/new404.html` doesn't look like any path from the docs either. Possibly this *is* website-related?

**Comment by jrodewig** [comment]
Thanks @gtback. I'll transfer this over to the webteam's repo.

One thing to note: Although my browser is set to the `FR` locale, the example above does go to our `en` docs (`https://www.elastic.co/guide/en/...`). It seems like something about the browser locale is stopping the custom 404.
